vividus/src/main/java/org/vividus/bdd/steps/BddVariableSteps.java

```diff
@@ -169,7 +158,15 @@ else if (instanceOfMap(variable1) && instanceOfMap(variable2))
                 return compareListsOfMaps(List.of(variable1), List.of(variable2));
             }
         }
-        return compare(variable1.toString(), condition, variable2.toString());
+        String variable1AsString = variable1.toString();
```

Comment:
I believe Sting.valueOf will be safer here?

Reply:
in general - agree
as for this case: do you have idea when `variable1`/`variable2` can be `null`?